src/components/NavBar/MainNavbar/RightMenu.js

```diff
@@ -1,21 +1,32 @@
 import React from "react";
-import { Menu } from "antd";
 import { useFirebase } from "react-redux-firebase";
 import { signOut } from "../../../store/actions";
-import { Avatar } from "antd";
+import Avatar from "@material-ui/core/Avatar";
 import { useAllowDashboard } from "../../../helpers/customHooks";
 import { useDispatch, useSelector } from "react-redux";
-import {
-  UserOutlined,
-  CodeOutlined,
-  LogoutOutlined,
-  BlockOutlined,
-  SettingOutlined
-} from "@ant-design/icons";
+import BlockOutlinedIcon from "@material-ui/icons/BlockOutlined";
+import SettingsOutlinedIcon from "@material-ui/icons/SettingsOutlined";
+import CodeOutlinedIcon from "@material-ui/icons/CodeOutlined";
+import ExitToAppOutlinedIcon from "@material-ui/icons/ExitToAppOutlined";
+import PersonOutlineOutlinedIcon from "@material-ui/icons/PersonOutlineOutlined";
```

Comment:
I'm not sure this is a good thing to do. Having more imports for icons makes code messier. Can't we follow the method we used to follow here?

Reply:
it was discussed in https://github.com/scorelab/Codelabz/pull/75#discussion_r638543643. So I was following this method.